rest-service/manager_rest/rest/resources_v3_1/labels.py

```diff
@@ -0,0 +1,21 @@
+from manager_rest import utils
+from manager_rest.storage import models
+from manager_rest.storage.models_base import db
+from manager_rest.security import SecuredResource
+from manager_rest.security.authorization import authorize
+
+
+class DeploymentsLabels(SecuredResource):
+
+    @authorize('labels_list')
+    def get(self):
+        """Get all deployments labels' keys"""
+        current_tenant = utils.current_tenant._get_current_object()
```

Comment:
You _get_current_object here but not in the other PR- why?

Reply:
I dropped it in the other PR for both classes based on @tehasdf explanation: 
```
utils.current_tenant is a proxy to a Tenant object, right
and normally getting attributes via the proxy, gets them from the underlying object
and ._get_current_object() dereferences the proxy, ie. just returns you the underlying object directly
it's important if you want to pass it to a different thread, because the proxies are thread-local
but in this case it does mostly nothing
```